.github/workflows/go.yml

```diff
@@ -0,0 +1,27 @@
+name: CI - base
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  code-checks:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.19
+
+    - name: Vet
+      run: go vet ./...
```

Comment:
should the Makefile target vet be used instead here?

Reply:
it's better indeed, fixing